packages/core/src/schema/index.ts

```diff
@@ -1,18 +1,13 @@
 export { latitudeSchema } from './db-schema'
-export {
-  promptVersions,
-  promptVersionRelations,
-  type PromptVersion,
-} from './models/promptVersions'
-export {
-  promptSnapshots,
-  promptSnapshotsRelations,
-  type PromptSnapshot,
-} from './models/promptSnapshots'
-export { commits, commitRelations, type Commit } from './models/commits'
+export { promptVersions } from './models/promptVersions'
+export { promptSnapshots } from './models/promptSnapshots'
+export { commits } from './models/commits'
 export { convos, convoRelations, type Convo } from './models/convos'
 export { users, type User } from './models/users'
 export { accounts, type Account } from './models/accounts'
 export { sessions, type Session } from './models/sessions'
 export { workspaces, type Workspace } from './models/workspaces'
 export { apiKeys, type ApiKey } from './models/apiKeys'
+
+export * from './models/types'
+export * from './models/relations'
```

Comment:
there were several circular dependencies in our models. This is why this exists.